<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MystackClass</class>
 <widget class="QMainWindow" name="MystackClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>It's MyStack!!!!!</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ari.jpg</normaloff>ari.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;输入表达式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="input_str">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>51</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>输入时请紧凑输入，不要出现空格       输入样例：6+8/(4-2)*3    (a+b)/(c-d)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>491</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>491</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Data Structure 24winter by AriHB</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="output_str">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>51</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;转化之后的表达式&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="b2">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>111</height>
       </size>
      </property>
      <property name="text">
       <string>后缀表达式转化为中缀表达式</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="b1">
      <property name="minimumSize">
       <size>
        <width>231</width>
        <height>111</height>
       </size>
      </property>
      <property name="text">
       <string>中缀表达式转化为后缀表达式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mystack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>b2</sender>
   <signal>clicked()</signal>
   <receiver>MystackClass</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b1</sender>
   <signal>clicked()</signal>
   <receiver>MystackClass</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button1_clicked()</slot>
  <slot>button2_clicked()</slot>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
 </slots>
</ui>
